Update Ruleset Engine concepts (#1603)
cloudflare/cloudflare-docs: products/firewall/src/content/cf-rulesets/custom-rulesets/add-rules-ruleset.md

# Add rules to a custom ruleset

To add rules to an existing custom ruleset, use the [Update ruleset](/cf-rulesets/rulesets-api/update) API method and pass the rules in an array. Each rule contains an expression and action.

<Aside type='note' header='Choosing the appropriate API method'>

> When you add rules to a ruleset using the [Update ruleset](/cf-rulesets/rulesets-api/update) method, you replace all the rules in the ruleset with the rules in the request. Use this API method when adding or updating several rules at once. It updates the ruleset version number only once.

Reviewer: ```suggestion
When you add rules to a ruleset using the [Update ruleset](/cf-rulesets/rulesets-api/update) method, you replace all the rules in the ruleset with the rules in the request. Use this API method when adding or updating several rules at once. This method updates the ruleset version number only once.
```

> You can use other API methods depending on the operation you wish to perform:

Reviewer: ```suggestion
You can use other API methods depending on the type of operation:
```
